.coderabbit.yaml

```
@@ -0,0 +1,132 @@
+      - code_duplication
+      - naming_conventions
+
+  # AI-powered code metrics
+  code_metrics:
+    enabled: true
+    analyze:
+      - cyclomatic_complexity
+      - cognitive_complexity
+      - maintainability_index
+      - code_duplication
+      - test_coverage
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)